docs/DeveloperGuide.md

```diff
@@ -216,6 +216,57 @@ The following sequence diagram shows how the show operation works.
  **Note:** The lifeline for `FilterInterviewCommandParser`
 should not exceed the destroy marker X. This is a known limitation of PlantUML.</div>
 
+
+### Find feature
+The ```find``` command is facilitated by creating a ```FindCommand``` depending on the given
+input. This command then updates the ```model``` accordingly.
+
+The following activity diagram summarizes what happens when a user executes a ```find``` command:
+
+![images](images/FindCommandActivityDiagram.png)
+
+Given below is an example usage scenario illustrated by a sequence diagram for ```find``` command.
+
+Step 1. A valid command `find n/Alex y/0` is given as user input. This invokes `LogicManager#execute()`, which calls
+`AddressBookParser#parseCommand()` to parse `find n/Alex y/0` into command word `find` and command argument ` n/Alex y/0`.
+
+Step 2. `FindCommandParser` is initialized based on the parse results and `FindCommandParser#parse()` is called
+to identify the predicates present in ` n/Alex y/0`. `FindCommandParser#parse()` then initializes a
+`FindCommand` with the predicates as argument.
+
+Step 3. `FindCommand#execute()` is then called, which will in turn call `Model#updateFilteredPersonList()`
+and filters for applicants that have `Alex` in their names and `0` year of experience.
+
+Step 4. Once the string of all applicant names is formed, `CommandResult` is initialized with this string as argument
+and returned.
+
+![images](images/FindCommandSequenceDiagram.png)
+
+<div markdown="span" class="alert alert-info">:information_source:
+ **Note:** The lifeline for `FindCommandParser`
+should not exceed the destroy marker X. This is a known limitation of PlantUML.</div>
+
+
+
+### Datetime for interview 
+The `Interview` class accepts `y-M-d, H:m` as parsed time format and provides `MMM dd yyyy , HH:mm` as display format.
+- `y` : year-of-era, e.g. `2021`, `21`
```

Comment:
If I'm not wrong, you changed the format for year the only accept 4 digits? But here you say it accepts 2 digits as well

Reply:
Ohh forgot to update, thanks!